craft_providers/multipass/multipass.py

```diff
@@ -159,7 +159,14 @@ def is_supported_version(self) -> bool:
         minimum_version = packaging.version.parse(self.minimum_required_version)
         version, _ = self.version()
 
-        return packaging.version.parse(version) >= minimum_version
+        parsed_version = None
+        while parsed_version is None:
+            try:
+                parsed_version = packaging.version.parse(version)
+            except packaging.version.InvalidVersion:
+                version = version.rpartition(".")[0]
```

Comment:
Can you add a comment here along the lines of `# X.Y.Z-dev.A.prB. isn't semvar compliant` ?

Also, can you update the comment block in `version()` to describe this new version syntax (line 493)?

Reply:
Technically they are semver compliant, but they're not PEP 440 compliant. I've added a comment to that effect though :-)